### PR TITLE
Better handle misconfigurations by stopping the bot

### PR DESCRIPTION
**Pull request by wagner-certat**
Two bugs are fixed:

* Handle non existing src or dest queues
  Currently, a bot without destination queue is retrying forever, now it stops immediately with a ConfigurationError
* bots handle invalid keys in incoming messages
  If an incoming message does contain an key/value which is not valid, we are retrying too, now it stops immediately with a ConfigurationError

### CODE REVIEW

**Comment by wagner-certat** [comment]
continuous-integration/travis-ci/push fails because of a lookup error

File: intelmq/bin/intelmqctl.py [hunk 1]
```diff
@@ -653,7 +653,7 @@ def get_queues(self):
     def list_queues(self):
         source_queues, destination_queues, internal_queues, all_queues = self.get_queues()
         pipeline = PipelineFactory.create(self.parameters)
-        pipeline.set_queues(source_queues, "source")
+        pipeline.set_queues(None, "source")
```
Comment (SYNchroACK) [inline intelmq/bin/intelmqctl.py]: did not understood this line

Comment (wagner-certat) [inline intelmq/bin/intelmqctl.py]: We need to set at least a dummy value here, otherwise we would fail later.

File: intelmq/lib/bot.py [hunk 2]
```diff
@@ -343,8 +352,13 @@ def receive_message(self):
...
 
-        self.__current_message = libmessage.MessageFactory.unserialize(message,
-                                                                       harmonization=self.harmonization)
+        try:
+            self.__current_message = libmessage.MessageFactory.unserialize(message,
+                                                                           harmonization=self.harmonization)
+        except exceptions.InvalidKey as exc:
+            # In case a incoming message is malformed an does not conform with the currently
+            # loaded harmonization, stop now as this will happen repeatedly without any change
+            raise exceptions.ConfigurationError('harmonization', exc.args[0])
```
Comment (aaronkaplan) [inline intelmq/lib/bot.py]: Which bots breaks now because of this ?

Comment (wagner-certat) [inline intelmq/lib/bot.py]: This is a better error handling (clear message for the user) in case of misconfigurations.

**Comment by wagner-certat** [comment]
No feedback - merged

**Pull request merged**